Source/Core/DolphinQt/Styles/Dark/dark.qss

```diff
@@ -485,3 +485,13 @@ QTabBar QToolButton::left-arrow {
 QTabBar QToolButton::right-arrow {
 	image: url(:/dolphin_dark_win/right-triangle-tabbar.svg);
 }
+/*
+Couldn't get this to work when writing it normally.
```

Comment:
I'm not sure what exactly you mean there?

Reply:
I couldn't do border-right: stuff. I had to do the whole thing then remove the other three sides.

Reply:
Ah, gotcha. Yeah Qt stylesheets can be weird with that; I think what happens internally is that the default you get if you set nothing and the default you get if you set something are different defaults, which can be rather confusing.

Reply:
Ah, if it's not too odd, should I remove the comment?

Reply:
Yes please.